docs/docs/guides/adding-command-line-completion.md

```diff
@@ -1,39 +1,45 @@
 ---
-title: Adding Tab Completion
-sidebar_label: Adding Tab Completion
+title: Adding Command-Line Completion
+sidebar_label: Adding Command-Line Completion
 ---
 
 <!-- NOTE TO KURTOSIS DEVS: 
 
-This page was generated by referencing the kubectl docs:
+This page was created by referencing the kubectl docs:
 * https://kubernetes.io/docs/tasks/tools/included/optional-kubectl-configs-bash-linux/
 * https://kubernetes.io/docs/tasks/tools/included/optional-kubectl-configs-bash-mac/
 * https://kubernetes.io/docs/tasks/tools/included/optional-kubectl-configs-zsh/
 * https://kubernetes.io/docs/tasks/tools/included/optional-kubectl-configs-fish/
 
 -->
 
-[The Kurtosis CLI](../reference/cli/cli.md) supports tab completion for `bash`, `zsh`, and `fish`. With tab completion installed, you will be able to:
+<!---------- START IMPORTS ------------>
 
-- Tab-complete subcommands (e.g. typing `kurtosis` and pressing TAB will suggest subcommands)
-- Tab-complete dynamic arguments, when possible (e.g. typing `kurtosis enclave inspect` and pressing TAB will try to complete with the current enclave IDs)
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-The process for installing tab completion is specific to each shell:
+<!---------- END IMPORTS ------------>
 
-<details>
-<summary>bash</summary>
+[The Kurtosis CLI](../reference/cli/cli.md) supports command-line completion for `bash`, `zsh`, and `fish`. With completion installed, you will be able to:
 
-1. Ensure you have Bash version >= 4.1
-    1. Print your Bash version:
+- Complete subcommands (e.g. typing `kurtosis` and pressing TAB will suggest subcommands)
+- Complete dynamic arguments (e.g. typing `kurtosis enclave inspect` and pressing TAB will list the names of existing enclaves if any)
```

Comment:
```suggestion
- Complete dynamic arguments (e.g. typing `kurtosis enclave inspect` and pressing TAB will list the names of running enclaves)
```

I believe this is technically *more* correct.

Reply:
Tab-complete will complete any enclave name, not just running ones (try `enclave stop` then `enclave inspect` tab-complete)